Scripts/Miscellaneous/Motion Detection and Tracking/requirements.txt

```diff
@@ -0,0 +1,2 @@
+python
```

Comment:
add specific module version

use 

`pip freeze list>requirements.txt`

Reply:
@chavarera , I have made the changes in requirements.txt file as per your request